roles/self_hosted_first_host/defaults/main.yml

```diff
@@ -27,6 +27,8 @@ config_dir: /etc/ansible-ovirt
 host_id: 1
 host_name: hosted_engine_{{ host_id }}
 compatibility_version: 4.0
+# product stand for decide to install ovirt/rhv appliance
+product: rhv
```

Comment:
Could you make the product/package defaults take into account the OS, since most people on centos will be doing ovirt